extensions/test-src/org/pentaho/platform/osgi/KarafBootTest.java

```diff
@@ -124,7 +123,13 @@ public void testBuildExtraKettleEtc() throws Exception {
 
   @Test
   public void testConfigureSystemProperties() throws Exception {
-
+    System.setProperty( "karaf.home", "karaf.home" );
+    String solutionRootPath =  "solutionRootPath";
```

Comment:
Shouldn't the test look at the other system.properties ?

karaf.data, for instance, would be useful since we're manipulating it when the instance starts.